client/components/Subhead.jsx

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Subhead = () => (
-    <div className= "subhead">Subhead</div>
+const Subhead = ({ condition, sold, availability }) => (
+    <div className= "subhead">
+        Subhead
+        {condition}
+        <br></br>
+        {sold}
+        <br></br>
+        {availability}
+    </div>
 );
 
+Subhead.propTypes = {
+  condition: PropTypes.bool.isRequired,
```

Comment:
Is the condition really going to be a boolean and not a string? Just confirming here, I think a boolean for used/new would be fine.

Reply:
Yeah the way our data was setup it is going be a boolean. True meaning free shipping, false meaning.. not free. lol